core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -3739,6 +3744,11 @@ definitions:
       useCallCache:
         type: boolean
         description: Whether or not to read from cache for this submission
+      deleteIntermediateOutputFiles:
+        type: boolean
+        description: >-
+          Whether or not to delete intermediate output files when the workflow completes.
+          See Cromwell docs for more information.
```

Comment:
If the Cromwell docs exist already, can we link them?

Reply:
Will research adding links into swagger/openapi. This "go find the docs urself" text was copypasta'd verbatim from below, but your suggestion would be a welcome improvement.

Reply:
Found it. Apparently one can embed markdown-in-swagger-yaml.

However, I will file a child ticket to add http links to the original "See docs" comment, this copy on this line, and any others "see" references that I find in the yaml.

Will also submit the PR for the ticket later tonight. Merging for this as is for now.

Reply:
As promised: https://github.com/broadinstitute/rawls/pull/1173